.changeset/tiny-jeans-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-manual-tests': patch
+---
+
+Add tests
```

Comment:
this changeset should be empty.

Reply:
Added correct changeset in this [commit](https://github.com/finos/legend-studio/pull/1238/commits/c8be5fe540dfc421847415c2a35aee08beedb737)